in/index.md

```diff
@@ -97,21 +97,21 @@ title: Daftar Isi Bantuan
 - [XMP metadata support in JabRef](/in/XMP)
 
 
-## Fetching entries from the web
```

Comment:
Translation does not work like that here.

Please use https://github.com/JabRef/help.jabref.org/blob/gh-pages/in/localization_in.json. Replace `""` by `"your-translation"`.